### PR TITLE
Features/movie details

### PR DESCRIPTION
**Pull request by PawelOwiesek**
First attempt at the MovieDetails component, there may still be minor changes to the styles. All  conflicts resolved!

### CODE REVIEW

File: src/common/TileList/MovieDetailsTile/styled.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  font-size: 36px;
+  font-weight: 600;
+  line-height: 1.2;
+  margin: 0;
+`;
+
+export const Year = styled.span`
+  font-size: 22px;
+  font-weight: 400;
+  line-height: 1, 2;
```
Comment (DorotaZielinska) [inline src/common/TileList/MovieDetailsTile/styled.js:33]: here insted of 1, 2 corect is 1.2

File: src/common/TileList/MovieDetailsTile/styled.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+export const Description = styled.span`
+  font-size: 18px;
+  font-weight: 400;
+  line-height: 1.2;
+  color: ${({ theme }) => theme.color.stormGray};
+`;
+
+export const Production = styled.p`
+  margin: 0;
```
Comment (DorotaZielinska) [inline src/common/TileList/MovieDetailsTile/styled.js:37]: here is a gap px  between production & relase date or line-height 1.2

**Review by MirekSobota — APPROVED** [review]
Based on what we discussed, the CSS needs some improvements, but the main logic is working fine! The styling doesn't impact the rest of the application, so we can merge this branch with the main one and continue working on this version. We'll need to make all the necessary CSS changes before we can do another deployment. 
Great job with the logic so far!

**Comment by DorotaZielinska** [comment]
I agre with Mirek, logic is fine only poster use  src could be in defrent place, there is css parts to change, as size of height where is bacdrop, size of Raiting in TileDetailsMovie is diffrent then in bacdrop and responsive part as well.

**Review by DorotaZielinska — APPROVED** [review]

**Pull request merged**